### DISCUSSION TITLE
Please add the alphabetical order nessessity for many-to-many table to your documentation.

### DISCUSSION THREAD

**Discussion by chrissisura** [prisma/prisma#3710] [Q&A] [answered]
It took me forever to find anything on that topic

**Comment by ryands17** [comment, marked as answer]
Hey @chrissisura 👋 

It's already present [here](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-schema/relations#conventions-for-relation-tables-in-implicit-m-n-relations). You can also search for _many to many_ in the search that we have and get the same result.

**Reply by chrissisura** [reply]
You‘re right! I‘m sorry for the mistake

**Reply by ryands17** [reply]
No worries man!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
